inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -48,14 +50,15 @@ In order to estimate stunting in the surveyed population, we looked at Height fo
 
 * Stunting: HAZ score <-2;
 * Moderate stunting: HAZ score >=-3 and <-2; Severe stunting: HAZ score <-3.
-* Exclusion of z-scores from Observed mean SMART flags included: WHZ -3 to 3; HAZ -3 to 3; WAZ -3 to 3.
+* Exclusion of z-scores from Observed mean SMART flags included: WHZ -3 to 3; HAZ -3 to 3; (Weight for age Z-scores) WAZ -3 to 3.
+
 
 # Results
 
 ```{r}
 # read in data
 # you can use rio to read in tabular data (Excel/csv) files
-survey_data <- rio::import("<your_data>")
+survey_data <- rio::import("<your_data>", which = 2)
```

Comment:
can you create a data dictionary that mirrors the mortality and nutrition data dictionaries please. 
Then adapt the [gen_data.R](https://github.com/R4EPI/sitrep/blob/master/R/gen_data.R) function with whichever variables are not defined in your dictionary. 
Also please follow [this section](https://github.com/R4EPI/sitrep/pull/84/files#diff-723e2366a318bfb7fa3e8e153182469bR21) from the mortality survey for reading in data examples. 
Also add example of reading in a datadictionary so it can be viewed along side while coding.